spec/models/spree/shipment_spec.rb

```diff
@@ -30,25 +30,53 @@
   end
 
   describe '.exportable' do
-    context 'when capture_at_notification is false' do
+    context 'when capture_at_notification is false and export_canceled_shipments is false' do
       it 'returns ready shipments from complete orders' do
-        stub_configuration(capture_at_notification: false)
+        stub_configuration(capture_at_notification: false, export_canceled_shipments: false)
 
         ready_shipment = create(:order_ready_to_ship).shipments.first
-        create(:shipped_order).shipments.first
+        create(:order_ready_to_ship) { |o| o.shipments.first.cancel! }
+        create(:shipped_order)
 
         expect(described_class.exportable).to eq([ready_shipment])
       end
     end
 
-    context 'when capture_at_notification is true' do
+    context 'when capture_at_notification is true and export_canceled_shipments is false' do
       it 'returns non-canceled shipments from complete orders' do
-        stub_configuration(capture_at_notification: false)
+        stub_configuration(capture_at_notification: true, export_canceled_shipments: false)
 
-        pending_shipment = create(:order_ready_to_ship).shipments.first
+        ready_shipment = create(:order_ready_to_ship).shipments.first
+        shipped_shipment = create(:shipped_order).shipments.first
         create(:order_ready_to_ship) { |o| o.shipments.first.cancel! }
 
-        expect(described_class.exportable).to eq([pending_shipment])
+        expect(described_class.exportable).to eq([ready_shipment, shipped_shipment])
```

Comment:
Why is the shipped shipment also returned here? I thought this would only return ready shipments.

Reply:
Because `capture_at_notification`is true. It should return all states except `canceled` (unless `export_canceled_shipments` is true). That's how it worked before, right?

Reply:
@igorbp you're right, sorry 😅